### ISSUE TITLE
Twirpscript fails with `Error: EAGAIN`

### ISSUE THREAD

**Issue by 4nte** [tatethurston/TwirpScript#7]
When running twirpscript this error occurs randomly, with no apparent cause.

Twirpscript was working fine for a while, then suddenly I started getting this error out of a blue.
Sometimes I need to run twirpscript few times in a row before it succeeds without the error. As if I have to wait for some lock to release, but there should be none.

```
node:internal/fs/utils:344
    throw err;
    ^

Error: EAGAIN: resource temporarily unavailable, write
    at Object.writeSync (node:fs:884:3)
    at writeFileSync (node:fs:2159:26)
    at Object.<anonymous> (<PROJECT_DIR_REDACTED>/proto/node_modules/twirpscript/dist/compiler.js:29:24)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:79:12)
    at node:internal/main/run_main_module:17:47 {
  errno: -11,
  syscall: 'write',
  code: 'EAGAIN'
}

```

**Comment by 4nte** [comment]
Upon further investigation, it seems that the issue is caused by writing "large" buffers to _stdout_.

I tested this by removing half of my proto files and running twirpscript successfully 100% of the time.
When I add more proto files, twirpscript starts consistently failing with the error above.

Refactoring 
```javascript
// Fails with EAGAIN when buffer too large
writeFileSync(process.stdout.fd, response.serializeBinary());
``` 

```javascript
// Succeeds with large buffers
process.stdout.write(Buffer.from(response.serializeBinary().buffer))
```

**Comment by tatethurston** [comment]
Hey @4nte thanks for reporting this issue. What version of node are you using? Also, do you have an example repo that I could take a look at?

**Comment by 4nte** [comment]
I'm running node v16.11.0 on Manjaro 21.1.6

Here's an example repo.
https://github.com/4nte/twirpscript-eagain-example
 
Steps to reproduce:
1. Run twirpscript

**Comment by tatethurston** [comment]
@4nte Thanks so much for the repro. I've published v0.0.22 which resolves the issue in your example. Let me know if you run into anything else!
